src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/index.js

```diff
@@ -3,7 +3,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import App from './App';
+////#if (IndividualLocalAuth)
+//// Uncomment the lines below to register the generated service worker.
+//// By default create-react-app includes a service worker to improve the
+//// performance of the application by caching static assets. This service
+//// worker, can however try and handle paths for the identity UI, so it is
+//// disabled by default when Identity is being used.
+////
+////import registerServiceWorker from './registerServiceWorker';
```

Comment:
Do we definitely need to include a repeat of this comment block, and the conditional `import` here?

If it's possible just to *always* have the `import registerServiceWorker` line, and then only toggle whether it gets used, I think that would be better.

If you're forced to do this as-is due to ESLint rules or something, I'd consider suppressing that rule here.

Reply:
It's because of linting. We could not include the commented out import and just include the comment in the lower line and tell people to uncomment the import too in the comment?

Reply:
Yes, that sounds better than duplicating the comment.

Reply:
Done